software-engineering-policies/PairProgramming/PairProgrammingPolicy.md

```diff
@@ -1,12 +1,9 @@
 # Pair Programming Policy
 
-## ​Guide
+## Guide
 
-All teams **must** carry out pair programming as a standard activity in their work.
+Most development activities will benefit from the collaboration of two or more colleagues working together. This practise encourages creativity, enhances problem solving and promotes knowledge sharing between colleagues.
```

Comment:
As much as I agree with the what is being said, as a policy it isn't clear about what the teams have to do, and thus is not measurable/auditable.

Policies need to clearly say what must be done (i.e. pass/fail). A good policy will also provide some guidance on the "how" and "why", but need to be clear on the "what".

Reply:
Is that not covered by "Teams should consider each activity that it undertakes and apply paired working if possible and practical to do so"

Reply:
> Is that not covered by "Teams should consider each activity that it undertakes and apply paired working if possible and practical to do so"

Possibly, just doesn't feel easily auditable?